README.md

```diff
@@ -22,11 +22,18 @@ Currently this module:
 - Adds more fine grained controls for firewalls, so will want to set `slurm::manage_firewall: false` from treydock/slurm to `false`
 - Allows telegraf scripts to be deployed which collect Slurm metrics.
 
+
 ## Setup
 
-For a Slurm client:
+For a Slurm login node:
```

Comment:
What is a slurm login node? Is this basically a submit node?

Reply:
Yep, it's submit node as far as Slurm is concerned, so that's probably a better way to phrase it.